### PR TITLE
Google oauth with Golang

### PR DESCRIPTION
**Pull request by 44t4nk1**

### CODE REVIEW

File: golang/googleOauth/main.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+)
+
+func init() {
+	googleOauthConfig = &oauth2.Config{
+		RedirectURL:  "http://localhost:8080/callback",
+		ClientID:     "YOUR_CLIENT_ID",
+		ClientSecret: "YOUR_CLIENT_SECRET",
+		Scopes:       []string{"https://www.googleapis.com/auth/userinfo.email"},
+		Endpoint:     google.Endpoint,
+	}
```
Comment (DiptoChakrabarty) [inline golang/googleOauth/main.go:24]: We should use environment variables to load the client id and secret here , please change this

Comment (DiptoChakrabarty) [reply golang/googleOauth/main.go:24]: @44t4nk1 also mention in the issue that you are willing to contribute to google  golang oauth

Comment (44t4nk1) [reply golang/googleOauth/main.go:24]: Okay

**Comment by 44t4nk1** [comment]
Environment variables used

**Review by DiptoChakrabarty — APPROVED** [review]

**Pull request merged**